entries/n/nelnet.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Nelnet": {
+    "domain": "nelnet.com",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "documentation": "https://nelnet.com/faqs#two-factor-auth",
+    "recovery": "https://nelnet.com/trouble-logging-in",
+    "categories": [
+      "banking"
```

Comment:
If the service only handles loans, wouldn't that mean the finances category would be a better fit?

Reply:
Yes